src/components/Navbar/Navbar.jsx

```diff
@@ -2,23 +2,25 @@ import React from "react";
 
 import s from "./Navbar.module.css";
 
+import { NavLink } from "react-router-dom";
+
 const Navbar = () => {
   return (
     <nav className={s.nav}>
       <div className={`${s.item} ${s.active}`}>
-        <a href="#">Profile</a>
+        <NavLink to="/profile" className={s.activeLink}>Profile</NavLink>
```

Comment:
Consider creating a folder "core" at the root level (e.g. src/core) and a "constants.js" file in that folder.
In this file you will keep the common reusable constants of your project.
As for the current line, consider creating object "ROUTES" inside the "constants.js" file and put "PROFILE" as a key and the "/profile" string as value.
Do this for EVERY such route in the whole project - find them and put them inside ROUTES object under the respective name.

Reply:
Done